packages/cdk/lib/cloudwatch-logs-management.test.ts

```diff
@@ -10,3 +12,16 @@ describe('The CloudwatchLogsManagement stack', () => {
 		expect(template.toJSON()).toMatchSnapshot();
 	});
 });
+
+
+// If stack contains PII data it should have a retention period of max 14 days (See list of retention presets here:
+// https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html).
+describe('Stacks with containsPIIData set to true', () => {
+	it('have retentionInDays <= 14', () => {
+		stacks.forEach((stack) => {
+			if (stack.containsPIIData && stack.retentionInDays) {
+				expect(stack.retentionInDays <= 14).toBe(true);
```

Comment:
minor: this could be simplified w/Jest's [`toBeLessThanOrEqual` matcher](https://jestjs.io/docs/expect#tobelessthanorequalnumber--bigint):

```ts
expect(stack.retentionInDays).toBeLessThanOrEqual(14);
```